package.json

```diff
@@ -29,17 +29,22 @@
     "ot": "./bin/ot"
   },
   "scripts": {
-    "start": "mkdir -p build && bin/server.js",
+    "watch": "watchify src/opentype.js --standalone opentype --debug -o build/opentype.js -v",
+    "start": "mkdirp build && parallelshell \"npm run watch\" \"node ./bin/server.js\"",
     "test": "jshint . && jscs .",
     "browserify": "browserify src/opentype.js --bare --standalone opentype > dist/opentype.js",
     "uglify": "browserify src/opentype.js --bare --standalone opentype -g uglifyify > dist/opentype.min.js",
-    "dist": "npm run test && npm run browserify && npm run uglify"
+    "dist": "rimraf build && rimraf dist && mkdirp build && mkdirp dist && npm run test && npm run browserify && npm run uglify"
+  },
+  "dependencies": {
   },
-  "dependencies": {},
   "devDependencies": {
     "browserify": "^9.0.3",
     "jscs": "^1.12.0",
     "jshint": "^2.6.3",
+    "mkdirp": "^0.5.1",
+    "parallelshell": "^1.1.1",
```

Comment:
cross-platform parallel task runner, so we can watchify and run server.js at the same time